packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -147,6 +155,14 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
         return odspResolvedUrl;
     }
 
+    private removeNavParam(link: string): string {
```

Comment:
I wonder if we should delete all query params? (if they are there)

Reply:
No, the /shares api does support query params and they maybe imp part of url providing the info. Actually nav param is appended by host in all resolve request and is lengthy and should be removed.